jac-uk/apply
# Admin #2165 Bug: Exercise. Eligibility Information. ASC word limit messaging is illogical

## What's included?
Ensure the default word limit displays for Additional Selection Criteria is one hasn't been specified in Admin.

Closes jac-uk/admin#2165

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
- Go to the following page (you may need to login first): https://jac-apply-develop--pr1106-feature-admin-2165-a-dneqo8tc.web.app/apply/Vulmahzn4VtelQhVlJy6/statement-of-eligibility
- Click 'YES' for both sections which say 'Do you meet this requirement'
- Ensure that the word limit for the first one is 250 and the second one is 10

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work

## Additional context

<img width="1429" alt="Screenshot 2023-11-10 at 15 13 16" src="https://github.com/jac-uk/apply/assets/115651787/046bf099-2630-4837-881a-ed0d6e15d7e1">

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
